### PR TITLE
DnD big ItemStack; Show green slot highlighting overlay

### PR DESCRIPTION
**Pull request by slprime**
## Show small itemStack when DnD
|before|after|
|-------|-----|
| image | image |

## Highlighting existing items
image

### CODE REVIEW

**Comment by Caedis** [comment]
The reason I didnt show green and red is because of red/green color blindness.
Only show 1 or the other, but not both.

**Comment by Caedis** [comment]
We have decided, keep the red and green, but change the default to false for the config added in #458

**Comment by slprime** [comment]
image

`Item Overlay on ?-Hover` - have 3 values: `None, Icon, Background`
`Highlight Present Item` - `Yes, No`

`Item Overlay on ?-Hover` join 2 options in one. `JEI Style Item Overlay on ?-Hover` and `Highlight the entire slot instead of just showing an icon for the recipe overlay`

**Comment by slprime** [comment]
| icon | background |
|----|---|
| image | image |
| image | image |

File: src/main/java/codechicken/nei/recipe/GuiRecipe.java [hunk 1]
```diff
@@ -1010,8 +1010,10 @@ public void drawGuiContainerForegroundLayer(int mouseX, int mouseY) {
 
         }
 
-        final boolean drawItemPresence = NEIClientConfig.isJEIStyleItemPresenceOverlayVisible();
-        final boolean slotHighlight = NEIClientConfig.isSlotHighlightEnabled();
+        final int presenceOverlay = NEIClientConfig.itemPresenceOverlay();
+        final boolean highlightPresentItem = NEIClientConfig.isSlotHighlightPresent();
+
+        System.out.println("presenceOverlay: " + presenceOverlay);
```
Comment (Caedis) [inline src/main/java/codechicken/nei/recipe/GuiRecipe.java:1016]: Left this in

**Review by Caedis — APPROVED** [review]

**Pull request merged**